shared/translations/languages/de.json

```
@@ -996,8 +996,13 @@
     "Wallet Address": "Walletadresse"
   },
   "components/EmptyBalances": {
-    "No other tokens yet": "Noch keine weiteren Token",
-    "Get started by adding your tokens here in your wallet": "Fange an, indem du deine Token hier deiner Wallet hinzufügst"
+    "No LP tokens in portfolio": "Keine Liquiditätspool-Token im Portfolio",
+    "No Crypto in portfolio": "Keine Kryptowährungen im Portfolio",
+    "No dTokens in portfolio": "Keine dToken im Portfolio"
+  },
+  "components/EmptyPortfolio": {
+    "Empty portfolio": "",
```

Better to copy the label as the translation as the fallback so that it won't be blank if it merged

```suggestion
    "Empty portfolio": "Empty portfolio",
```